metadatastore/commands.py

```diff
@@ -839,6 +841,11 @@ def _transform_data(data, timestamps):
     to storage format:
         {'data': {<key>: (<value>, <timestamp>)}.
     """
+    # shallow copy so we can do some norrmalization
+    data = dict(data)
```

Comment:
Oh, is that what @stuwilkins is doing?  Putting the timestamps data in as a data key?